pkg/apb/dockerhub_registry.go

```diff
@@ -173,20 +169,78 @@ func (r *DockerHubRegistry) loadBundleImageData(org string) ([]*ImageData, error
 	}
 
 	channel := make(chan *ImageData)
+	ctx, cancelFunc := context.WithCancel(context.Background())
+	defer cancelFunc()
```

Comment:
This is going to kill all requests when ```loadBundleImageData``` returns. I think we either want a timeout context for pure async or we can synchronously load images 100 at a time.